src/test/loader.ts

```diff
@@ -437,3 +438,10 @@ const baseFullConfig: FullConfig = {
   workers: 1,
   launch: [],
 };
+
+function resolveScript(id: string, rootDir: string) {
+  const localPath = path.resolve(rootDir, id);
+  if (fs.existsSync(localPath))
+    return localPath;
+  return require.resolve(id, { paths: [rootDir] });
```

Comment:
So, this is going to work for `globalSetup: 'playwright-dev-server'` where you use a node module as a global setup?

Reply:
yes

Reply:
added a test to confirm